contracts/hydro/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "hydro"
-version = "1.0.0"
+version = "1.1.0"
```

Comment:
Should we also use version 1.1.0 for tribute contract? Currently it is set to 2.0.

Reply:
yes, probably right